felix/config/config_params.go

```diff
@@ -160,12 +160,13 @@ type Config struct {
 	DataplaneDriver            string `config:"file(must-exist,executable);calico-iptables-plugin;non-zero,die-on-fail,skip-default-validation"`
 
 	// Wireguard configuration
-	WireguardEnabled               bool   `config:"bool;false"`
-	WireguardListeningPort         int    `config:"int;51820"`
-	WireguardRoutingRulePriority   int    `config:"int;99"`
-	WireguardInterfaceName         string `config:"iface-param;wireguard.cali;non-zero"`
-	WireguardMTU                   int    `config:"int;0"`
-	WireguardHostEncryptionEnabled bool   `config:"bool;false"`
+	WireguardEnabled               bool          `config:"bool;false"`
+	WireguardListeningPort         int           `config:"int;51820"`
+	WireguardRoutingRulePriority   int           `config:"int;99"`
+	WireguardInterfaceName         string        `config:"iface-param;wireguard.cali;non-zero"`
+	WireguardMTU                   int           `config:"int;0"`
+	WireguardHostEncryptionEnabled bool          `config:"bool;false"`
+	WireguardPersistentKeepAlive   time.Duration `config:"seconds;25"`
```

Comment:
We may want this to be a pointer so that we can default to the current behavior - is there any chance users will want to opt-out of this behavior?

Reply:
From a quick search, I think we will need to make this disabled by default and an opt-in feature - I suspect most wireguard users will want this disabled unless using a NAT / etc.

Reply:
If a value of 0 disables the keepalive, should 0 be the default?

Reply:
Yeah, we should update this to 0 and we should change the FelixConfiguration API documentation to indicate that the field is default off.

Reply:
Changed default value to 0